include/alpaka/idx/bt/IdxBtOacc.hpp

```diff
@@ -0,0 +1,128 @@
+/* Copyright 2020 Axel Huebl, Jeffrey Kelling, Benjamin Worpitz, René Widera
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#ifdef ALPAKA_ACC_ANY_BT_OACC_ENABLED
+
+#if _OPENACC < 201306
+    #error If ALPAKA_ACC_ANY_BT_OACC_ENABLED is set, the compiler has to support OpenACC xx or higher!
+#endif
+
+#include <alpaka/idx/Traits.hpp>
+#include <alpaka/workdiv/Traits.hpp>
+#include <alpaka/workdiv/WorkDivMembers.hpp>
+
+#include <alpaka/vec/Vec.hpp>
+#include <alpaka/core/Positioning.hpp>
+#include <alpaka/core/Unused.hpp>
+#include <alpaka/idx/MapIdx.hpp>
+#include <alpaka/core/Concepts.hpp>
+
+namespace alpaka
+{
+    namespace bt
+    {
+        //#############################################################################
+        //! The OpenACC accelerator ND index provider.
+        template<
+            typename TDim,
+            typename TIdx>
+        class IdxBtOacc : public concepts::Implements<ConceptIdxBt, IdxBtOacc<TDim, TIdx>>
```

Comment:
According to the naming we used for OpenMP (`IdxGbLinear`), this should be called `IdxBtLinear`.